.mergify.yml

```diff
@@ -1,61 +1,75 @@
 queue_rules:
   - name: default
-    conditions:
-      - status-success=build
+    merge_method: squash 
+    queue_conditions:
```

Comment:
From https://docs.mergify.com/configuration/file-format/#queue-rules it looks like we don't need to duplicate queue_conditions in merge_conditions.  From https://docs.mergify.com/merge-queue/setup/#configuring-the-merge-queue-rules it looks like we only need one.  I'd probably stick with queue_conditions and ditch merge_conditions.